docs/en/engines/table-engines/special/distributed.md

```diff
@@ -163,6 +163,8 @@ Clusters are configured in the [server configuration file](../../../operations/c
         <shard>
             <!-- Optional. Shard weight when writing data. Default: 1. -->
             <weight>1</weight>
+            <!-- Optional. The shard number. If not specified, numbers will be generated automatically. -->
```

Comment:
I would even add a note that using `shard_number` is recommended for production clusters

Reply:
Let's add there that it's 1-based and should be unique among shards in the cluster

Reply:
>Let's add there that it's 1-based and should be unique among shards in the cluster
Okey

> I would even add a note that using shard_number is recommended for production clusters
Should we add this one?

Reply:
IMHO, any manual non synchronized change to config can lead to a mess. People can make mistakes by assigning inconsistent shard numbers as well. So, using shard number and not using them, both approaches have their own flaws. I would still consider the shard number as some internal thing.